go.mod

```diff
@@ -152,6 +151,7 @@ require (
 	k8s.io/kms v0.27.2 // indirect
 	k8s.io/kube-aggregator v0.26.1 // indirect
 	sigs.k8s.io/cluster-api-provider-aws/v2 v2.0.2 // indirect
+	sigs.k8s.io/cluster-api-provider-ibmcloud v0.2.4 // indirect
```

Comment:
Just curious, was it needed to move that here?

Reply:
wasnt needed, just a side effect of me hacking at this so many times